### DISCUSSION TITLE
How deal with URI?

### DISCUSSION THREAD

**Discussion by sanpii** [todotxt/todo.txt#48] [Q&A]
Hello,

It’s easy to ignore URL (ie `https://`) from keywords but how can we (parser writers) should deal with the others URI like `mailto:`, `tel:`, [etc.](https://en.wikipedia.org/wiki/Category:URI_schemes)?

**Comment by tomByrer** [comment]
Don't include the URI?

**Comment by tensau** [comment]
> Don't include the URI?

That's not a solution.

**Comment by TangentFoxy** [comment]
@sanpii Why do you want to ignore it? It's a valid key:value pair, just of an unusual nature. Personally I prefer to just accept protocols as commonly occurring key:value tags.

**Comment by alswl** [comment]
This request need multi line(or note feature) in spec.
The URL is title will be simple text type, or maybe the wrapper App will treat the URL as URL(can click and open it in browser).

**Comment by xuhcc** [comment]
The parser can simply ignore key/value tag if the key is not whitelisted. Typically applications only support a limited number of keys such as `due`, `rec` or `h`

**Comment by AaronKnowls** [comment]
If using sleek as a middle-man for todo.txt, Markdown works for URIs while allowing you to click them.

Example:
(E) do this and that [uri]((obsidian://open?vault=vault&file=inbox)
